java/src/main/native/src/ScalarJni.cpp

```diff
@@ -321,7 +334,7 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_Scalar_makeStringScalar(JNIEnv *env,
       strval.assign(reinterpret_cast<char *>(jbytes.data()), jbytes.size());
     }
 
-    auto s = new cudf::string_scalar{strval, is_valid};
+    auto s = new cudf::string_scalar{strval, static_cast<bool>(is_valid)};
```

Comment:
Curious, why do we need to static cast a `jboolean` parameter here but not in other cases like line 317 of this file or [here](https://github.com/rapidsai/cudf/blob/branch-0.15/java/src/main/native/src/TableJni.cpp#L1726).

Reply:
The curly brace initialization (aka list initialization) does less implicit type conversion. I guess gcc-9 is more strict about it now.